install/helm/agones/templates/crds/fleetautoscaler.yaml

```diff
@@ -107,6 +107,13 @@ spec:
                         caBundle:
                           type: string
                           format: byte
+                        fallback:
+                          type: object
+                          nullable: true
+                          properties:
+                            policy:
+                              type: object
+                              x-kubernetes-preserve-unknown-fields: true
```

Comment:
I'd rather we have an actual spec here.

My thought here would be to take the `webhook` element, and turn it into a Helm `include` or `template` - much like we do for [_gameserverstatus.yaml](https://github.com/googleforgames/agones/blob/main/install/helm/agones/templates/crds/_gameserverstatus.yaml) and use that in both spots.

The include will need a conditional though to make sure it only recurses one level, but since you can pass in a context structure that should be doable.